.github/actions/setup-solana/action.yaml

```diff
@@ -11,7 +11,7 @@ runs:
           ~/.cache/solana/
           ~/.local/share/solana/
         key: solana-${{ runner.os }}-v0000-${{ env.SOLANA_CLI_VERSION }}
-    - run: sh -c "$(curl -sSfL https://release.solana.com/v${{ env.SOLANA_CLI_VERSION }}/install)"
```

Comment:
If you put --retry 3 in `- run: sh -c "$(curl --retry 3 -sSfL https://release.solana.com/v${{ env.SOLANA_CLI_VERSION }}/install)"`. Is it working ?

Reply:
sure i'll try that

Reply:
There is --retry-all-errors, I don't know what is the best

Reply:
yeh, i just added that one as well. But i wonder if maybe it will need a time delay between retries, since we don't know exactly what's causing it.

Reply:
going to try using a native github actions retry instead since this curl retry doesn't appear to change anything